tools/privacyidea-migrate-linotp.py

```diff
@@ -173,182 +220,135 @@
 conn_linotp = linotp_engine.connect()
 conn_pi = privacyidea_engine.connect()
 
-resolver_map = {"LDAPIdResolver": "ldapresolver",
-                "SQLIdResolver": "sqlresolver",
-                "PasswdIdResolver": "passwdresolver"}
 
-# Process Tokens
+# Values to be imported
```

Comment:
I think we should be able to remove the definitions of ``linotp_config_table``, ``linotp_tokenrealm_table``, ``linotp_realm_table`` above.

Reply:
OK

Reply:
Removed tables